dockerfiles_for_OpenEthereum_CI/utility/README.md

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
for all these OE dockerfiles I think it would make sense to change metadata and test if they are working brfore moving away, but not in this PR.